### ISSUE TITLE
Data loss while streaming from characteristicValueStream

### ISSUE THREAD

**Issue by ext-danny** [rohitsangwan01/win_ble#6]
I was printing the value from the stream

` WinBle.characteristicValueStream.listen((event) {
       log('$event', name: '#characteristicValueStream');
    });`

There is data loss from the plugin. How do you propose to handle it?

**Comment by rohitsangwan01** [comment]
@ext-danny can you explain in detail , with sample like what you are getting and what was expected

**Comment by ext-danny** [comment]
So once I connect and subscribe I would  be sending a command to fetch some detail of a ble device which would be sending me  8 lines of data

**Expected output**

Data Format v1.0<\r><\n>
ID: Name<\r><\n>
1:  Sensor_1<\r><\n>
2: Sensor_2<\r><\n>
3: Sensor_3<\r><\n>
6: Sensor_6<\r><\n>
8: Sensor_8<\r><\n>
10: Sensor_10<\r><\n>

**Received Output**

Data Format v1.0<\r><\n>
ID: Name<\r><\n>
1:  Sensor_1<\r><\n>
6: Sensor_6<\r><\n>
8: Sensor_8<\r><\n>
10: Sensor_10<\r><\n>

This is just one example but on multiple tries the data loss is different. 
And at times I get only the first line when I send the command and the rest of the data when I send the command again.

**Comment by rohitsangwan01** [comment]
@ext-danny can you test on Android NrfConnect or any other Windows app , just to confirm

**Comment by ext-danny** [comment]
@rohitsangwan01  yep sure. it is working fine on Android I'm able to get the expected output but will also check on windows using other Windows app.

**Comment by ext-danny** [comment]
Hi @rohitsangwan01 is there any app that you would suggest to check on the windows cause like I said the android part works as expected every time.

**Comment by rohitsangwan01** [comment]
@ext-danny can you share complete Logs

**Comment by rohitsangwan01** [comment]
@ext-danny please try with latest version now V1.0.0+
